### DISCUSSION TITLE
Arthas redefine command invalid after using Skywalking Agent

### DISCUSSION THREAD

**Discussion by malversation** [apache/skywalking#9100] [Q&A]
Arthas Re-define command invalid after using Skywalking Agent，
The agent.config configuration file has been modified，
redefine the command executed successfully, but the class in the JVM did not change。

Skywalking OAP Server Version: 9.0.0

Skywalking Agent Version: 8.10.0

Arthas version: 3.5.1

**Comment by wu-sheng** [comment]
We are not experts about `Arthas`. AFAIK, `Arthas` is a runtime debug tool, and it will instrument/change codes after SkyWalking did. 
So, what would impact Arthas should be addressed by their team. If they have some questions, I could join to assist, but after all, we can't determine a runtime attached agent's behaviors.
